src/landingPage/SecuritySection.module.scss

```diff
@@ -20,17 +20,19 @@
   }
 
   .lock {
-    width: 10rem;
-    position: absolute;
-    // Center the lock in the middle of the screen
-    top: 50%;
-    left: 50%;
-    transform: translate(-50%, -50%);
-    z-index: 1;
-
+    padding-top: 5vh;
     img {
-      overflow: visible;
+      height: 30vh;
+      display: block;
+      margin: auto;
+      overflow: hidden;
     }
+
+    display: inline-block;
+    position: relative;
+    width: 100%;
+    padding-bottom: 100%; 
+    vertical-align: middle; 
```

Comment:
I don't think the vertical-align is needed, since the img on the inside fills the entire height of this div